x/fees/types/params.go

```diff
@@ -9,11 +9,16 @@ import (
 
 // Parameter store key
 var (
-	DefaultDeveloperShares       = sdk.NewDecWithPrec(50, 2) // 50%
-	DefaultValidatorShares       = sdk.NewDecWithPrec(50, 2) // 50%
-	ParamStoreKeyEnableFees      = []byte("EnableFees")
-	ParamStoreKeyDeveloperShares = []byte("DeveloperShares")
-	ParamStoreKeyValidatorShares = []byte("ValidatorShares")
+	DefaultEnableFees      = false
+	DefaultDeveloperShares = sdk.NewDecWithPrec(50, 2) // 50%
+	DefaultValidatorShares = sdk.NewDecWithPrec(50, 2) // 50%
+	// cost for `crypto.CreateAddress`
+	// keccak256(word) costs 36 gas
+	DefaultAddrDerivationCostCreate       = uint64(50)
```

Comment:
I'm a bit confused bc the comment says 36 gas but the default value is 50, can you clarify?

Reply:
Because the address does some additional operations aside from `keccak256`, I increased the cost. I will improve the comment.
https://github.com/ethereum/go-ethereum/blob/d8ff53dfb8a516f47db37dbc7fd7ad18a1e8a125/crypto/crypto.go#L108-L111
```
func CreateAddress(b common.Address, nonce uint64) common.Address {
	data, _ := rlp.EncodeToBytes([]interface{}{b, nonce})
	return common.BytesToAddress(Keccak256(data)[12:])
}
```